### PR TITLE
Update README.md to include a better downloads section

### PR DESCRIPTION
**Pull request by EXtremeExploit**
# Preview

## Downloads
* [Windows](https://github.com/Vita3K/Vita3K/actions?query=event%3Apush+is%3Asuccess+branch%3Amaster)
  * [Visual C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Install if not working.) 
* [Linux](https://github.com/Vita3K/Vita3K/actions?query=event%3Apush+is%3Asuccess+branch%3Amaster)
* [macOS](https://github.com/Vita3K/Vita3K/actions?query=event%3Apush+is%3Asuccess+branch%3Amaster)

Includes macOS, Linux and the C++ Runtime redistributables to avoid more vcruntime questions on the discord

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,11 @@ The emulator currently runs some homebrew programs. It is also able to load some
 
 Vita3K is licensed under the **GPLv2** license. This is largely dictated by external dependencies, most notably Unicorn.
 
-## Binaries
-
-Vita3K binaries for Windows can be downloaded on [AppVeyor](https://ci.appveyor.com/project/Vita3K/vita3k/branch/master/artifacts).
+## Downloads
+* [Visual Studio C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Requiered for windows build, install if not working)
```
Comment (Margen67) [inline README.md:40]:
```suggestion
* [Visual Studio C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Required for Windows build, install if not working.) 
```

File: README.md [hunk 2]
```diff
@@ -36,9 +36,11 @@ The emulator currently runs some homebrew programs. It is also able to load some
 
 Vita3K is licensed under the **GPLv2** license. This is largely dictated by external dependencies, most notably Unicorn.
 
-## Binaries
-
-Vita3K binaries for Windows can be downloaded on [AppVeyor](https://ci.appveyor.com/project/Vita3K/vita3k/branch/master/artifacts).
+## Downloads
+* [Visual Studio C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Required for Windows build, install if not working.)
```
Comment (ichisadashioko) [inline README.md:40]: It's just `Visual C++` (`Studio` is not included). 😂

**Comment by EXtremeExploit** [comment]
please review it and squash

File: README.md [hunk 3]
```diff
@@ -36,9 +36,11 @@ The emulator currently runs some homebrew programs. It is also able to load some
 
 Vita3K is licensed under the **GPLv2** license. This is largely dictated by external dependencies, most notably Unicorn.
 
-## Binaries
-
-Vita3K binaries for Windows can be downloaded on [AppVeyor](https://ci.appveyor.com/project/Vita3K/vita3k/branch/master/artifacts).
+## Downloads
+* [Visual C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Required for Windows build, install if not working.) 
+* [Windows](https://ci.appveyor.com/project/Vita3K/vita3k/branch/master/artifacts)
```
Comment (Margen67) [inline README.md:41]:
```suggestion
* [Windows](https://ci.appveyor.com/project/Vita3K/vita3k/branch/master/artifacts)
    * [Visual C++ Redistributables](https://aka.ms/vs/16/release/vc_redist.x64.exe) (Required for Windows build, install if not working.)
```

**Comment by ichisadashioko** [comment]
@EXtremeExploit you need to learn to use `git commit --amend`, `git reset --soft`, and `git push --force` then come back and commit your PR again.

**Pull request merged**